docs/operator-manual/applicationset/Generators-Git.md

```diff
@@ -382,6 +385,61 @@ spec:
 
 In `values` we can also interpolate all fields set by the git files generator as mentioned above.
 
+### Pass additional parameter files via `extraParameterFiles` field
+
+You may provide an additional list of files containing parameters (yaml or json format) with the `extraParameterFiles` of the git generator.
+Extra parameters added via the `extraParameterFiles` field are added as `extraParams.(field)`.
+
+These files are expected to be resolved in the same git repository as the rest of the generator.
+
+The files provided supports glob format, and you can use any parameter resolved from the files or directories fields to templatize the file path.
+An extra parameter file not found will not trigger an error, and will just be ignored.
+
+A hierarchy occurs in case of collision betweens the parameters, as follows:
+
+* Parameters are merged and overriden file by file, from top to bottom of the list provided
+* When a file provided is a glob pattern, files are sorted by path and overrides occur in the sorted order.
```

Comment:
I'm not 100% sure it's a good idea to provide globs here. 
Forcing people to sort alphabetically can lead to error. I would rather enforce explicit path

What do you think @crenshaw-dev @ishitasequeira ?

Reply:
I'm partial to leaving the glob supported as long as we document it properly people can use it and it's convenient.